src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,70 @@
+import classNames from 'classnames';
+import React from 'react';
+
+type Props = {
+  status: string;
+  onStatus: React.Dispatch<React.SetStateAction<string>>;
+  amountOfItems: number;
+};
+
+export const Footer: React.FC<Props> = ({
+  status,
+  onStatus,
+  amountOfItems,
+}) => {
+  const FILTERS = {
+    all: 'all',
+    completed: 'completed',
+    active: 'active',
+  };
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${amountOfItems} items left`}
+      </span>
+
+      <nav className="filter" data-cy="Filter">
```

Comment:
I was told to create a separate file for filter. Tell you in advance to do the same